package.json

```diff
@@ -48,12 +49,14 @@
     ]
   },
   "dependencies": {
+    "@types/lodash": "^4.14.121",
```

Comment:
From testing locally, it doesn't look like adding these to dependencies provides any additional benefit, but maybe I'm missing something?

Reply:
The reasoning is that by installing packages with types to dependencies, dependent packages using TypeScript won't have to reinstall them and can share the package version (as having two different versions of the same type package can cause type errors).

Source: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies

Personally I prefer to put type packages in the same category as the original package. Also TypeScript's docs show TypeScript itself in dependencies, the rationale is that while it is a development tool, it still includes types for production code.

Reply:
Interesting, thanks for the explanation!